<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Toolbar</class>
 <widget class="QWidget" name="Toolbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>30</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>48</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>27</red>
        <green>27</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>27</red>
        <green>27</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>27</red>
        <green>27</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Link">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>27</red>
        <green>27</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>27</red>
        <green>27</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>27</red>
        <green>27</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Link">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>27</red>
        <green>27</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>27</red>
        <green>27</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>27</red>
        <green>27</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Link">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget 
{

	border: none;
}


QToolButton
{
	border: 1px solid black;
    border-radius: 0px;
}

</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="pauseBtn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>30</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>40</width>
       <height>30</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="LinearGradientPattern">
          <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>79</red>
             <green>79</green>
             <blue>170</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>16</red>
             <green>16</green>
             <blue>51</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="LinearGradientPattern">
          <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>79</red>
             <green>79</green>
             <blue>170</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>16</red>
             <green>16</green>
             <blue>51</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="LinearGradientPattern">
          <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>79</red>
             <green>79</green>
             <blue>170</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>16</red>
             <green>16</green>
             <blue>51</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="128">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="LinearGradientPattern">
          <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>79</red>
             <green>79</green>
             <blue>170</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>16</red>
             <green>16</green>
             <blue>51</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="LinearGradientPattern">
          <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>79</red>
             <green>79</green>
             <blue>170</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>16</red>
             <green>16</green>
             <blue>51</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="LinearGradientPattern">
          <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>79</red>
             <green>79</green>
             <blue>170</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>16</red>
             <green>16</green>
             <blue>51</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="128">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>27</red>
           <green>27</green>
           <blue>43</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>27</red>
           <green>27</green>
           <blue>43</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>27</red>
           <green>27</green>
           <blue>43</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton:enabled {
    background-color: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #4F4FAA,  /* Top-left */
        stop:1 #101033     /* Bottom-right */
    );
}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::MediaPlaybackPause"/>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="playBtn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>30</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>40</width>
       <height>30</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="LinearGradientPattern">
          <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>0</red>
             <green>170</green>
             <blue>0</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>0</red>
             <green>51</green>
             <blue>0</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="LinearGradientPattern">
          <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>0</red>
             <green>170</green>
             <blue>0</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>0</red>
             <green>51</green>
             <blue>0</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="LinearGradientPattern">
          <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>0</red>
             <green>170</green>
             <blue>0</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>0</red>
             <green>51</green>
             <blue>0</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="128">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="LinearGradientPattern">
          <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>0</red>
             <green>170</green>
             <blue>0</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>0</red>
             <green>51</green>
             <blue>0</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="LinearGradientPattern">
          <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>0</red>
             <green>170</green>
             <blue>0</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>0</red>
             <green>51</green>
             <blue>0</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="LinearGradientPattern">
          <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>0</red>
             <green>170</green>
             <blue>0</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>0</red>
             <green>51</green>
             <blue>0</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="128">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>27</red>
           <green>27</green>
           <blue>43</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>27</red>
           <green>27</green>
           <blue>43</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>27</red>
           <green>27</green>
           <blue>43</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton:enabled {
	background-color: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #00AA00,  /* Top-left */
        stop:1 #003300     /* Bottom-right */
    );
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="popupMode">
      <enum>QToolButton::ToolButtonPopupMode::DelayedPopup</enum>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonStyle::ToolButtonIconOnly</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="stopBtn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>30</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>40</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton:enabled {
	background-color: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #9F0000,  /* Top-left */
        stop:1 #300000     /* Bottom-right */
    );
}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::MediaPlaybackStop"/>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="recordBtn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>30</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>40</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton:checked {
	background-color: qlineargradient(x1:0,y1:0,x2:1,y2:1, stop:0 #AA3000, stop:1 #200500); 
}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::MediaRecord"/>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
